### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by straend**
Use Official PHP image with Apache.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,21 +1,28 @@
-FROM tutum/apache-php
-MAINTAINER Sam Stenvall <neggelandia@gmail.com>, Richard Weber <riche.weber@gmail.com>
+FROM php:7.1-apache
+LABEL maintainer="Sam Stenvall <neggelandia@gmail.com>, Richard Weber <riche.weber@gmail.com>, Tomas Strand <tomas@fik1.net>"
 
 ENV ALLOW_OVERRIDE=true
+ENV APACHE_DOCUMENT_ROOT /app
+RUN sed -ri -e 's!/var/www/html!${APACHE_DOCUMENT_ROOT}!g' /etc/apache2/sites-available/*.conf
+RUN sed -ri -e 's!/var/www/!${APACHE_DOCUMENT_ROOT}!g' /etc/apache2/apache2.conf /etc/apache2/conf-available/*.conf
```
Comment (Jalle19) [inline Dockerfile]: Maybe it would be better to use /var/www instead of /app so these files don't need to be changed?

File: Dockerfile [hunk 2]
```diff
@@ -1,21 +1,28 @@
...
+
 #Git is good for composer
 RUN apt-get update && \
-    apt-get install -yq git php5-imagick php5-sqlite php5-json && \
+    apt-get install -yq git mercurial unzip imagemagick magickwand-6.q16-dev && \
```
Comment (Jalle19) [inline Dockerfile]: Why is mercurial needed?

File: Dockerfile [hunk 3]
```diff
@@ -1,21 +1,28 @@
...
 RUN rm -fr /app
 
 ADD . /src
 
 WORKDIR /src
-RUN composer install
-RUN ./src/protected/yiic createinitialdatabase
-RUN ./src/protected/yiic setpermissions
+RUN curl -sS https://getcomposer.org/installer | php
+RUN php composer.phar install --no-plugins --no-scripts
```
Comment (Jalle19) [inline Dockerfile]: Did you check that Composer isn't installed already in the image?

File: Dockerfile (hunk 1 above)
Comment (straend) [reply Dockerfile]: Yes, it would, did a change

File: Dockerfile (hunk 2 above)
Comment (straend) [reply Dockerfile]: Not really, I missed it, it was needed before unzip was installed, removed.

File: Dockerfile (hunk 3 above)
Comment (straend) [reply Dockerfile]: Yes, it's not.

**Pull request merged**